src/Http/Role/AddLabelToRoleRequestHandler.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Role;
+
+use Broadway\CommandHandling\CommandBus;
+use CultuurNet\UDB3\Http\ApiProblem\ApiProblem;
+use CultuurNet\UDB3\Http\Request\RouteParameters;
+use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
+use CultuurNet\UDB3\Model\ValueObject\Identity\UUID;
+use CultuurNet\UDB3\Role\Commands\AddLabel;
+use Fig\Http\Message\StatusCodeInterface;
+use InvalidArgumentException;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use Slim\Psr7\Response;
+
+class AddLabelToRoleRequestHandler implements RequestHandlerInterface
+{
+    private CommandBus $commandBus;
+
+    private ReadRepositoryInterface $labelRepository;
+
+    public function __construct(CommandBus $commandBus, ReadRepositoryInterface $labelRepository)
+    {
+        $this->commandBus = $commandBus;
+        $this->labelRepository = $labelRepository;
+    }
+
+    public function handle(ServerRequestInterface $request): ResponseInterface
+    {
+        $routeParameters = new RouteParameters($request);
+        $roleId = $routeParameters->getRoleId();
+
+        try {
+            $roleId = new UUID($roleId);
+        } catch (InvalidArgumentException $e) {
+            throw ApiProblem::roleNotFound($roleId);
+        }
+
+        $labelIdentifier = $routeParameters->get('labelIdentifier');
+        try {
+            $labelId = new UUID($labelIdentifier);
+        } catch (InvalidArgumentException $exception) {
+            $entity = $this->labelRepository->getByName($labelIdentifier);
+
+            if ($entity === null) {
+                throw ApiProblem::blank('There is no label with identifier: ' . $labelIdentifier, 404);
```

Comment:
Is this `::blank()` type recently added? The `about:blank` API problem type may only be used in some specific cases where the error type can be deduced from the status code (for example 502 = always bad gateway, no further explanation needed). But even then it is allowed to use a more specific type than `about:blank`, and definitely in this case it should be more specific.

Given that this error is caused by a URL parameter, the type should probably be `::urlNotFound()`. I know it's a bit weird in this case because the route to handle this request exists, and the role also exists. But it's because the label name is sent as part of the URL instead of being a part of the body that makes this a weird situation.

Reply:
cc @LucWollants @JonasVHG to avoid any confusion/assumptions about the `about:blank` API problem type

Reply:
The `::urlNotFound()` makes sense imo. I'll update it.